src/hooks/protections/script_event_handler.cpp

```diff
@@ -335,6 +337,19 @@ namespace big
 			}
 			break;
 		}
+		case eRemoteEvent::InteriorControl:
+			int interior = (int)args[2];
+			if (interior < 0 || interior > 158) // the upper bound will change after an update
```

Comment:
Can these bounds be easily found by using the event hash? If not, we something that can help us find the upper bound when the game updates.